frontend/src/types/NMPFileGeneratedManureData.ts

```diff
@@ -10,5 +10,6 @@ interface NMPFileGeneratedManureData {
   IsMaterialStored?: boolean;
   AssignedToStoredSystem?: boolean;
   AssignedWithNutrientAnalysis?: boolean;
+  manureId: string | null;
```

Comment:
Should this be allowed to be null? That would imply there's no animal.

Reply:
I want the field to be mandatory, but there's moment the default is generated where there's no id.